.goreleaser.yml

```diff
@@ -1,18 +1,22 @@
 # This is an example .goreleaser.yml file with some sane defaults.
 # Make sure to check the documentation at http://goreleaser.com
+
```

Comment:
why do we need this env variable, if we have the same in the `builds` ?

Reply:
leftover...I will remove the one from the builds. thanks!